apps/microsoft-teams/app-actions/src/types.ts

```diff
@@ -28,6 +28,9 @@ export enum AppEventKey {
   ENTRY_DELETED = 'ContentManagement.Entry.delete',
   ENTRY_ARCHIVE = 'ContentManagement.Entry.archive',
   ENTRY_UNARCHIVE = 'ContentManagement.Entry.unarchive',
```

Comment:
In the same ballpark as [this comment](https://github.com/contentful/apps/pull/6888/files#r1529519858), This feels like a code smell, seeing as a workflow update (workflow event?  I kinda prefer update.  Sounds more like a state change, which is the model I think they're going for.  IMO.) is not the same as an entry event. (publish, create etc.)